examples/pytorch/README.md

```diff
@@ -0,0 +1,42 @@
+# PyTorch data loading with Mountpoint
+
+This directory shows a simple example of how to use Mountpoint for Amazon S3 as a PyTorch data loader via the torchdata library.
+It trains a ResNet-50 model for a few epochs using [PyTorch Lightning](https://www.pytorchlightning.ai/index.html), with synthetic ImageNet-style training data, stored in S3 as shards in WebDataset format.
+The idea is that because Mountpoint exposes your S3 bucket as a local file system, you can just use standard file-oriented loaders rather than writing S3-specific code, and get great performance.
+
+To get started, first install the dependencies (we recommend using the [AWS Deep Learning AMI for PyTorch](https://aws.amazon.com/machine-learning/amis/)):
+
+    pip install -r requirements.txt
+
+Now install Mountpoint if you don't already have it:
+
+    wget https://s3.amazonaws.com/mountpoint-s3-release/latest/x86_64/mount-s3.rpm
+    sudo yum install ./mount-s3.rpm
+
+To generate and upload the training shards to an S3 bucket, run:
+
+    python resnet.py make s3://DOC-EXAMPLE-BUCKET/shard-data/ --num-images 50000
```

Comment:
I got a `ModuleNotFoundError` complaining about `certifi` module when running this command. Adding this module to the requirements file solved the problem for me. I think we need to update yours too.